### PR TITLE
docs: sms migration guide

### PR DESCRIPTION
**Pull request by manchuck**
## Description
SMS migration docs

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+If you would like to use this SDK as a standalone package, you can install it using:
+
+```console
+$ npm install @vonage/sms
+```
+
+## Setup
+
+### With the Vonage Server SDK
+If you are using the main Vonage Node Server SDK, you will need to configure it using a Vonage API Key and Secret. The setup is the same for any other key/secret-based application. The applications client will then be available as `vonage.sms`.
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]:
```suggestion
If you are using the main Vonage Node Server SDK, you will need to configure it using a Vonage API Key and Secret. The setup is the same for any other key/secret-based application. The SMS client will then be available as `vonage.sms`.
```

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+```js
+const Vonage = require('@vonage/server-sdk');
+
+const vonage = new Vonage({
+    applicationId: APP_ID,
+    privateKey: PRIVATE_KEY_PATH,
+});
+
+vonage.sms.listCountryPricing()
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]: Should be an example to send an SMS

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Setup
+
+### With the Vonage Server SDK
+If you are using the main Vonage Node Server SDK, you will need to configure it using a Vonage API Key and Secret. The setup is the same for any other key/secret-based application. The applications client will then be available as `vonage.sms`.
+
+```js
+const Vonage = require('@vonage/server-sdk');
+
+const vonage = new Vonage({
+    applicationId: APP_ID,
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]: This should be a key/secret auth instead

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    privateKey: PRIVATE_KEY_PATH,
+});
+
+vonage.sms.listCountryPricing()
+    .then(resp => console.log(resp))
+    .catch(err => console.error(err));
+```
+
+### Standalone
+This module is also capable of working as a standalone module. The only difference is that you create an `Applications` object and pass the credentials there.
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]:
```suggestion
This module is also capable of working as a standalone module. The only difference is that you create a `SMS` object and pass the credentials there.
```

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Changed Methods
+
+| 2.x Method | 3.x Method | Notes |
+|----------------|---------------|-------|
+| `vonage.sms.sendSms()` | `vonage.sms.send()` | This method has been renamed and now takes in a `SMSParams` object instead of individual parameters |
+| `vonage.sms.sendBinaryMessage()` | _Removed_ |  This has been removed |
+| `vonage.sms.sendWapPushMessage()` | _Removed_ |  This has been removed |
+| `vonage.sms.search()` | _Removed_ |  This has been removed |
+| `vonage.sms.searchREjections()` | _Removed_ |  This has been removed |
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]:
```suggestion
| `vonage.sms.searchRejections()` | _Removed_ |  This has been removed |
```

**Review by dragonmantank — CHANGES_REQUESTED** [review]
Couple of typos and need a valid example

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+```js
+const Vonage = require('@vonage/server-sdk');
+
+const vonage = new Vonage({
+    apKey: API_KEY,
+    apiSecret: API_SECRET,
+});
+
+vonage.sms.({
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]: This is missing the method name :P

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
+# Migration guide from Vonage Node SDK 2.x to 3.x
+
+## Installation
+
+The package name has changed and is now a companion to the core Vonage Node SDK. You can continue to use the `@vonage/server-sdk` package or now use the Numbers SDK as a standalone package. If you would like to continue to use the full SDK, update the version number to `^3.0` and follow this migration guide.
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]: Should say SMS and not Numbers (I think I missed this the first pass)

**Review by dragonmantank — CHANGES_REQUESTED** [review]

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md (hunk 6 above)
Comment (manchuck) [reply packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]: this is what happens when you don't have your normal keyboard and muck up your vim commands

File: packages/sms/v2_TO_v3_MIGRATION_GUIDE.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+## Setup
+
+### With the Vonage Server SDK
+If you are using the main Vonage Node Server SDK, you will need to configure it using a Vonage API Key and Secret. The setup is the same for any other key/secret-based application. The SMS client will then be available as `vonage.sms`.
+
+```js
+const Vonage = require('@vonage/server-sdk');
+
+const vonage = new Vonage({
+    apKey: API_KEY,
```
Comment (dragonmantank) [inline packages/sms/v2_TO_v3_MIGRATION_GUIDE.md]:
```suggestion
    apiKey: API_KEY,
```

**Review by dragonmantank — CHANGES_REQUESTED** [review]

**Review by dragonmantank — APPROVED** [review]

**Pull request merged**